types/humanize-duration/index.d.ts

```diff
@@ -140,6 +140,10 @@ declare namespace humanizeDuration {
          * Customize the value used to calculate each unit of time.
          */
         unitMeasures?: UnitMeasuresOptions | undefined;
+        /**
+         * Array of ten strings to which will replace the numerals 0-9.
+         */
+        digitReplacements?: string[] | undefined;
```

Comment:
This works but doesn't protect against arrays of length != 9. Consider:

```suggestion
        digitReplacements?: Record<"0" | "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9", string> | undefined;
```

See [the JSDoc types in HumanizeDuration.js](https://github.com/EvanHahn/HumanizeDuration.js/blob/392c869c8f7db7f359ab9011e05a8f32a9e5cf66/humanize-duration.js#L26) for more.

Reply:
[The docs](https://www.npmjs.com/package/humanize-duration#digitreplacements) are referring it as an array so specify it as `Record` might be a bit confusing IMO.

Maybe the docs should be updated or specify the type as:
```typescript
digitReplacements?: [string, string, string, string, string, string, string, string, string, string] | undefined;
```

WDYT?

Reply:
Good point. I've [updated HumanizeDuration's type][0] based on your suggestion. Let's go with that!

[0]: https://github.com/EvanHahn/HumanizeDuration.js/commit/83ff8534f1e7916a8bedb585a3bbfb23e1683327

Reply:
Done :)